pkg/oci/utils_test.go

```diff
@@ -86,18 +86,24 @@ func TestMinimalPodConfig(t *testing.T) {
 			Destination: "/proc",
 			Type:        "proc",
 			Options:     nil,
+			IgnoreMount: false,
+			HostPath:    "",
 		},
 		{
 			Source:      "tmpfs",
 			Destination: "/dev",
 			Type:        "tmpfs",
 			Options:     []string{"nosuid", "strictatime", "mode=755", "size=65536k"},
+			IgnoreMount: false,
+			HostPath:    "",
 		},
 		{
 			Source:      "devpts",
 			Destination: "/dev/pts",
 			Type:        "devpts",
 			Options:     []string{"nosuid", "noexec", "newinstance", "ptmxmode=0666", "mode=0620", "gid=5"},
+			IgnoreMount: false,
+			HostPath:    "",
 		},
 	}
 
```

Comment:
@mcastelino  I though /dev/shm needs to be created inside the VM, is that right. I am planning to send all the mount info for the mounts that need to be created within the VM (including /dev/shm) so that hyperstart can create those. Planning to send a separate PR for this.(for virtcontainers and hyperstart)